storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -14,7 +14,7 @@ import README from "../../../../packages/css/src/components/field-set/README.md?
 
 ## Examples
 
-### With description
+### With Description
```

Comment:
We chose sentence case for headings in #1565. Only component names in headings still start with uppercase letters, but `Description` is not a component.

```suggestion
### With description
```